### ISSUE TITLE
Undefined symbols for architecture armv7

### ISSUE THREAD

**Issue by trimaximus** [Tencent/MMKV#847]
## Note:
Uses `xcodebuild` to archive the project leads to `Undefined symbols for architecture armv7`

### The language of MMKV

> swift

### The version of MMKV

> v1.2.12  

### The platform of MMKV

> iOS

### The installation of MMKV

> Cocoapods

### What's the issue?

When I used `xcodebuild` to archive my project:
```shell
xcodebuild archive -workspace XXX.xcworkspace -scheme XXX.Test -archivePath ./build/archive/2022-03-10-13-53-08.xcarchive -configuration Release
```

The Terminal showed:
```shell
Undefined symbols for architecture armv7:
  "_crc32", referenced from:
      mmkv::MMKV::checkFileCRCValid(unsigned long, unsigned int) in MMKV.o
      mmkv::MMKV::recaculateCRCDigestWithIV(void const*) in MMKV.o
      mmkv::MMKV::updateCRCDigest(unsigned char const*, unsigned long) in MMKV.o
      mmkv::MMKV::partialLoadFromFile() in MMKV_IO.o
      mmkv::MMKV::isFileValid(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >*) in MMKV_IO.o
ld: symbol(s) not found for architecture armv7
clang: error: linker command failed with exit code 1 (use -v to see invocation)

** ARCHIVE FAILED **


The following build commands failed:
	Ld /Users/trimaximus/Library/Developer/Xcode/DerivedData/SISYPHE-czhaugjxqgunmremibndrprbsiao/Build/Intermediates.noindex/ArchiveIntermediates/SISYPHE.Test/IntermediateBuildFilesPath/Pods.build/Release-iphoneos/MMKVCore.build/Objects-normal/armv7/Binary/MMKVCore normal armv7 (in target 'MMKVCore' from project 'Pods')
(1 failure)

```

**Comment by trimaximus** [comment]
Then I changed my architectures to arm64, still get the same error.

**Comment by lingol** [comment]
I don't have any clue. Have you tried to archive the MMKVDemo project in this repo? Does it produce the same error?

**Comment by trimaximus** [comment]
MMKVDemo was built successfully.
I found that in MMKVDemo, the `Link Binary with Libraries` of `MMKVCore` contains `libz.tbd`
![Demo](https://user-images.githubusercontent.com/9427799/157611702-f98320ff-e666-4bbd-a502-fbf515c57ef1.png)
But in my `Cocoapods` , `libz.tbd` is missing
![My Project](https://user-images.githubusercontent.com/9427799/157611984-2cf44b14-7fa7-4840-aa5e-25d558df44b5.png)
But I can still archive my project with Xcode. When I added `libz.tbd` in `MMKVCore` mannually, the build was successed.
That may by the reason to cause this problem.
That's really weird to me.
